<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationDialog</class>
 <widget class="QDialog" name="RegistrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>648</width>
    <height>436</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>648</width>
    <height>436</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Full name</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="firstName">
        <property name="placeholderText">
         <string>First name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="secondName">
        <property name="placeholderText">
         <string>Second name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="patronymic">
        <property name="placeholderText">
         <string>Patronymic</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Login data</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="login">
        <property name="placeholderText">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>User type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="userGroup">
        <item>
         <property name="text">
          <string>Administrator</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dispatcher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Worker</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="professionsGroup">
     <property name="title">
      <string>Professions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="btnAddProfession">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QTableView" name="userProfessions">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnRemoveProfession">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnSetProfessionCurrent">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Set current</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>630</width>
       <height>23</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RegistrationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RegistrationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddProfession</sender>
   <signal>clicked()</signal>
   <receiver>RegistrationDialog</receiver>
   <slot>addNewProfession()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemoveProfession</sender>
   <signal>clicked()</signal>
   <receiver>RegistrationDialog</receiver>
   <slot>removeOldProfession()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSetProfessionCurrent</sender>
   <signal>clicked()</signal>
   <receiver>RegistrationDialog</receiver>
   <slot>setCurrentProfession()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addNewProfession()</slot>
  <slot>removeOldProfession()</slot>
  <slot>setCurrentProfession()</slot>
 </slots>
</ui>
